charts/datadog/ci/agent_otel_collector.yaml

```diff
@@ -0,0 +1,24 @@
+targetSystem: "linux"
+datadog:
+  apiKey: "00000000000000000000000000000000"
+  appKey: "0000000000000000000000000000000000000000"
+  otelCollector:
+    enabled: true
+    config:
+      receivers:
+        otlp:
+      exporters:
+        datadog:
+          api:
+            key: "00000000000000000000000000000000"
+      service:
+        pipelines:
+          traces:
+            receivers: [otlp]
+            exporters: [datadog]
+          metrics:
+            receivers: [otlp]
+            exporters: [datadog]
+          logs:
+            receivers: [otlp]
+            exporters: [datadog]
```

Comment:
Sorry if its not documented, but the file name need to finish with -values.yaml to be taken into account. (like any other files in this folder)